includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -581,16 +581,15 @@ public function get_title() {
 		$title = parent::get_title();
 
 		if (
-			Payment_Method::CARD === $this->stripe_id &&
```

Comment:
The "Test mode" badge is now displayed on all methods, when test mode is enabled. It'll be displayed by CSS only when the payment method is selected.